smartlisten.lic

```diff
@@ -10,23 +10,25 @@ no_kill_all
 
 listen_skills = get_settings.listen_skills
 
+listen_skills += ['Life Magic', 'Elemental Magic', 'Lunar Magic', 'Holy Magic'] unless DRStats.barbarian? || DRStats.thief?
```

Comment:
❓ Does listening to a cross-magic-type, which teaches Sorcery, cause problems in town?

Reply:
no, only if you listen to a sorcery class. listening to elemental magic as a life magic user teaches sorcery instead of elemental magic without consequence

Reply:
I have been doing this for over a year inadvertently since life magic is included in the default listen_skills: in base.yaml for everyone, that's why I was surprised it was only life magic being listened to.

Reply:
💡 I think we should probably remove Life Magic from `base.yaml`